text/2952-futures.md

```diff
@@ -59,52 +59,35 @@
 
 自從最初的 futures 0.3 發布以來，除了上面提到的改進之外，核心 `Future` 特徵和任務系統的變化相對較小。實際的 `Future` 特徵基本上與 4 月時相同。
 
-# Guide-level explanation
-[guide-level-explanation]: #guide-level-explanation
+# 教學式解說
+[教學式解說]: #guide-level-explanation
 
-The `Future` trait represents an *asynchronous* and lazy computation that may
-eventually produce a final value, but doesn't have to block the current thread
-to do so.
-
-Futures can be constructed through `async` blocks or `async` functions, e.g.,
+`Future` 特徵代表了一個非同步和惰性計算，它最終可能會產生一個最終值，但不以阻塞當前執行緒為前提來達成。
 
+Future 可以通過 `async` 區塊或 `async` 函式來創建，例如
 ```rust
 async fn read_frame(socket: &TcpStream) -> Result<Frame, io::Error> { ... }
 ```
 
 This `async` function, when invoked, produces a future that represents the
 completion of reading a frame from the given socket. The function signature
 is equivalent to:
+這個 `async` 函式，當被呼叫時，產生一個 future，代表從给定的 socket 中讀取一個幀的完成。函式簽名等同於：
```

Comment:
will do (possibly tomorrow) 🙇

Reply:
I will keep going to translate the third stage.
Have any recommendations, I will be back here.

Reply:
In Traditional Chinese, we call "frame" is "框" https://terms.naer.edu.tw/search/?q=%22frame%22&field=ti&op=AND&match=full&q=&field=ti&op=AND&order=&num=10&show=&page=&group=&heading=#result
But I think this will make reader confuse
So, I put the English followed by the word, which I think is more appropriate